e2e/cypress/integration/team_settings/create_a_team_spec.js

```diff
@@ -52,4 +45,55 @@ describe('Teams Suite', () => {
         // * Team name should displays correctly at top of LHS
         cy.get('#headerTeamName').should('contain', 'Team Test');
     });
+
+    it('MM-T1437 Try to create a new team using restricted words', () => {
+        openTeamCreationModal()
+
+        // # Input team name as Team Test
+        cy.get('#teamNameInput').should('be.visible').type('plugins');
+
+        // # Click Next button
+        cy.findByText('Next').should('be.visible').click();
```

Comment:
Could you please move these and the last step to close the modal to `tryReservedTeamURLAndVerifyError`?